parm/config/config.fcst

```diff
@@ -88,7 +88,7 @@ else
 fi
 
 # Radiation options 
-export IAER=5111    ;#spectral band mapping method for aerosol optical properties
+export IAER=1111    ;#spectral band mapping method for aerosol optical properties
```

Comment:
Please add a reviewer to this PR who can check the science-based changes, particularly the change to IAER and the diag_table below. Thanks!